src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -107,22 +107,24 @@
 
 "ExposureDetection_Guide_Hands" = "Waschen Sie Ihre Hände regelmäßig mit Seife für 20 Sekunden.";
 
-"ExposureDetection_Guide_Mask" = "Tragen Sie eine Maske bei Begegnungen mit anderen Personen.";
+"ExposureDetection_Guide_Mask" = "Tragen Sie eine Maske bei Begegnungen mit anderen Personen in Innenräumen und dort, wo es gesetzlich vorgeschrieben ist oder empfohlen wird.";
 
-"ExposureDetection_Guide_Distance" = "Halten Sie mindestens 1,5 Meter Abstand zu anderen Personen.";
+"ExposureDetection_Guide_Distance" = "Halten Sie möglichst mindestens 1,5 Meter Abstand zu anderen Personen.";
 
 "ExposureDetection_Guide_Sneeze" = "Niesen oder husten Sie in die Armbeuge oder in ein Taschentuch.";
 
 "ExposureDetection_Guide_Ventilation" = "Lüften Sie Innenräume mehrmals täglich. Öffnen Sie dazu die Fenster einige Minuten lang so weit wie möglich (Stoßlüftung).";
 
-"ExposureDetection_Guide_Symptoms" = "Achten Sie auf Krankheitssymptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten. Falls Sie sich krank fühlen oder Symptome auftreten, wenden Sie sich bitte an Ihre Hausarztpraxis.";
+"ExposureDetection_Guide_Symptoms" = "Achten Sie auf Krankheitssymptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten. Falls Sie sich krank fühlen oder Symptome auftreten, reduzieren Sie Ihre Kontakte, bleiben Sie zu Hause und wenden Sie sich an Ihre Hausarztpraxis.";
 
-"ExposureDetection_Guide_Home" = "Begeben Sie sich, wenn möglich, nach Hause bzw. bleiben Sie zu Hause und vermeiden Sie Begegnungen außerhalb Ihres Haushalts.";
+"ExposureDetection_Guide_ReduceContacts" = "Wenn es Ihnen möglich ist, reduzieren Sie bitte Ihre Kontakte, vor allem mit besonders gefährdeten Personen, z.B. Menschen hohen Alters, mit Vorerkrankungen oder einem geschwächten Immunsystem";
```

Comment:
```suggestion
"ExposureDetection_Guide_ReduceContacts" = "Wenn es Ihnen möglich ist, reduzieren Sie bitte Ihre Kontakte, vor allem mit besonders gefährdeten Personen, z.B. Menschen hohen Alters, mit Vorerkrankungen oder einem geschwächten Immunsystem.";
```

Reply:
yes in Figma the dot is missing as well. Good catch!